osu.Game.Rulesets.Osu/UI/Cursor/GameplayCursor.cs

```diff
@@ -190,5 +206,15 @@ private void calculateScale()
                 cursorContainer.Scale = new Vector2(scale);
             }
         }
+
+        private class SkinnableCursor : SkinReloadableDrawable
```

Comment:
On second check, this is completely wrong. You're only using this drawable to look up the value.

Maybe `OsuCursor : SkinReloadableDrawable` and put it there? Or make this called CursorScaleContainer at very least (since it's not a `SkinnableCursor`).